test/e2e-api/admin/offers.test.js

```diff
@@ -0,0 +1,527 @@
+const {agentProvider, fixtureManager, matchers} = require('../../utils/e2e-framework');
+const {anyEtag, anyObjectId, anyLocationFor, anyErrorId} = matchers;
+const should = require('should');
+const models = require('../../../core/server/models');
+
+let agent;
+
+async function getPaidProduct() {
+    return await models.Product.findOne({type: 'paid'});
+}
+
+async function getFreeProduct() {
+    return await models.Product.findOne({type: 'free'});
+}
+
+describe('Offers API', function () {
+    let defaultTier;
+    let savedOffer;
+
+    before(async function () {
+        agent = await agentProvider.getAdminAPIAgent();
+        await fixtureManager.init();
+        await agent.loginAsOwner();
+        defaultTier = await getPaidProduct();
+    });
+
+    it('Has no initial offers', async function () {
+        await agent
+            .get(`offers/`)
+            .expectStatus(200)
+            .matchHeaderSnapshot({
+                etag: anyEtag
+            })
+            .matchBodySnapshot({
+                offers: []
```

Comment:
You only need to supply an object to matchBodySnapshot if you need to pass matchers for values that change like IDs and Dates.